src/Util/Logger/SyslogLogger.php

```diff
@@ -0,0 +1,297 @@
+<?php
+
+namespace Friendica\Util\Logger;
+
+use Friendica\Network\HTTPException\InternalServerErrorException;
+use Friendica\Util\Strings;
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\LoggerInterface;
```

Comment:
This is why I don't like PSR-3 at all: An interface is an interface, no need to add `Interface` to it's name. Like you have to add `Class` to each class ...

Reply:
Same as above ;-)